containerfiles/Containerfile.ci

```diff
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi9/ubi-init:latest
+ENV USE_VENV=no
+
+RUN dnf update -y && dnf install -y git python3-pip make gcc sudo && yum clean all
+
+COPY ../ /opt/sources
```

Comment:
I think that should work in Prow, but we will only know after merging and testing.
In order to have this working, you will need to start from pipeline:src, which will have the code for you to COPY.

Reply:
I've already prepared the update for patch against release - we'll see how it goes once we get this one in. A pity we can't Depends-On....